From 92510738b9987aa6ef9a2000367868ec8edce58a Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Fri, 25 Mar 2022 11:25:22 -0300
Subject: [PATCH 030/163] posix: Remove unused variable on tst-_Fork.c

Checked on x86_64-linux-gnu and i686-linux-gnu.

(cherry picked from commit f902892aaf92295a7d3065370412fb51a13e13e9)
---
 posix/tst-_Fork.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/posix/tst-_Fork.c b/posix/tst-_Fork.c
index d383f87bf7..ae1ba9bb21 100644
--- a/posix/tst-_Fork.c
+++ b/posix/tst-_Fork.c
@@ -85,7 +85,6 @@ singlethread_test (void)
 }
 
 
-static volatile sig_atomic_t sigusr1_handler_ran;
 #define SIG_PID_EXIT_CODE 20
 
 static bool atfork_prepare_var;
-- 
2.37.2

